### ISSUE TITLE
[Toggle button] Buttons on documentation website do not have accessible names

### ISSUE THREAD

**Issue by a11ymiko** [sl-design-system/components#3110]
## Current behavior 

Toggle buttons examples at documentation website ([Usage](https://sanomalearning.design/categories/components/toggle-button/usage/) and [Code](https://sanomalearning.design/categories/components/toggle-button/code/)) don't have any accessible names provided using `aria-label`, `aria-labelledby` or invisible text. 
Screen reader user placing focus on that elements gets announcement: "toggle button" that is not sufficient.

## Suggested solution

It's recommended to use solution that @michal-sanoma  was working on in Storybook. That includes providing button's name with `aria-label` before tooltip is added to DOM. After tooltip is added to DOM accessible name should be provided using `aria-labelledby` and tootip's `id`. 

### 👤 Your name

Kacper

### 🧢 Your Product/Team

SLDS
